src/ArcValidation/ExpectoExtensions.fs

```diff
@@ -72,6 +72,12 @@ module Expecto =
     /// Generate test results using in a minimal JUnit schema.
     let writeJUnitSummary file (summary: Impl.TestRunSummary) =
```

Comment:
LGTM, but i'd suggest maybe adding a `verbose` argument so we can return the full error stack if we really need it.